add C8Run documentation to self managed (#4266)
camunda/camunda-docs: docs/self-managed/setup/deploy/local/c8run.md

---
id: c8run

> title: "Local installation with C8Run"

Reviewer: Can we use Camunda 8 Run (C8Run). C8Run looks like an abbreviation. It will also help with SEO.

Reviewer 2: Late to the party 💃 but here I am!! 😆 

> Can we use Camunda 8 Run (C8Run). C8Run looks like an abbreviation. It will also help with SEO.

@conceptualshark you can consider using `keywords` in the page metadata for things like this.

```
keywords: ["C8Run"]
```

When successful, a new Operate window will open automatically.

:::note
If C8Run fails to start, run the [shutdown script](#shut-down-c8run) to end the current processes before running the start script again.
:::

### Configuration options

The following command line arguments are available:

| Argument     | Description                                                                                                                                  |
| ------------ | -------------------------------------------------------------------------------------------------------------------------------------------- |
| `--config`   | Applies the specified Zeebe [`application.yaml`](/self-managed/zeebe-deployment/configuration/configuration.md). _Not available on Windows._ |
| `--detached` | Starts C8Run as a detached process. The process is detached by default on Windows.                                                           |

## Access Camunda components

All C8Run components can be accessed with the username/password combination `demo`/`demo`.

Tasklist and Operate are available at:

- Tasklist: `http://localhost:8080/tasklist`
- Operate: `http://localhost:8080/operate`

> The following components do not have a web interface, but the URLs may be required for any additional configuration:

Reviewer 3: ```suggestion
The following components do not have a web interface, but the URLs may be required for additional configuration:
```

- Zeebe Gateway: `http://localhost:26500`
- Connectors: `http://localhost:8085`

:::note
The Connectors URL displays a login page, but cannot be logged into.
:::

### Deploy diagrams from Desktop Modeler

To [deploy diagrams](/self-managed/modeler/desktop-modeler/deploy-to-self-managed.md) from Desktop Modeler, the following configuration is required:

- **Target:** Self-Managed
- **Cluster endpoint:** `http://localhost:26500`, the location of your Zeebe Gateway
- **Authentication:** None

> A success notification will display when complete. [Start a new process instance](/components/modeler/desktop-modeler/start-instance.md) to view your running process in Operate.

Reviewer 3: ```suggestion
A success notification is shown when deployment is complete. [Start a new process instance](/components/modeler/desktop-modeler/start-instance.md) to view your running process in Operate.
```
